src/components/AreaSelect/leaflet-areaselect.css

```diff
@@ -0,0 +1,15 @@
+.leaflet-areaselect-shade {
```

Comment:
Can you move this file into the `src/styles/vendor/` directory? Note that you'll also need to import it in `src/styles/index.css`